### PR TITLE
Closes #193 updated inappropriate PDu values

### LINKED ISSUE #193: Feature Request: Update rs_onco_pcwg
### Feature Idea

The `rs_onco_pcwg` dataset should be updated such that the value `PDu` for bone lesion response can occur only at the last assessment. If it is not at the last assessment it should be updated to `PD` or `NON-PD`.

The combined response should be updated accordingly.

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by tomyukn**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [X] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [X] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [X] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [X] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [X] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [X] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [X] Address any updates needed for vignettes and/or templates
- [X] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [X] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [X] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [X] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [X] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/rs_onco_pcwg3.R [hunk 1]
```diff
@@ -19,7 +19,7 @@ pcwg3_raw <- tibble::tribble(
   "01-701-1034", "WEEK 16",     "PD",        "SD", "NON-PD",     "SD",
   "01-701-1034", "WEEK 24",     "PD",        "SD", "NON-PD",     "SD",
   "01-701-1097",  "WEEK 8",     "PR",        "SD", "NON-PD",     "SD",
-  "01-701-1097", "WEEK 16",     "PR",        "PR",    "PDu",     "PR",
+  "01-701-1097", "WEEK 16",     "PR",        "PR", "NON-PD",     "PR",
```
Comment (bundfussr) [inline data-raw/rs_onco_pcwg3.R]: I think this is not correct. As PD requires confirmation, you can't have a single PD. Either it should be PD at WEEK 16 or it should be PDu at WEEK 24.

Do you agree?

File: NEWS.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 -  Updated QSTESTCD in `qs_ophtha` to be consistent with CDISC terminology. Also reduced the number of records for ease of use. (#172)
 
+- Updated `rs_onco_pcwg3` to correct the misclassification of `PDu`. (#193)
```
[the remark this answers is not in the document]
Comment (tomyukn) [reply NEWS.md]: Updated the NEWS.md item.
